src/aap_eda/api/serializers/user.py

```diff
@@ -102,6 +103,12 @@ class Meta:
             "is_superuser",
         ]
 
+    def validate_is_superuser(self, value):
```

Comment:
I don't see this getting called anywhere, and while testing I was able to update `superuser`'s username with a normal user, which I don't think is the expectation here. Maybe it should be renamed to `validate()` or get called in `override()` method?

Reply:
Ah, that could be a bug in that I put this on `UserCreateUpdateSerializer` but not `CurrentUserUpdateSerializer` or the other serializers. I would prefer to put this on a base user serializer.

But the pattern of `validate_{field_name}` is a DRF pattern. I tested this for one case and it is effective. I'd like to keep this method, because I think the issue is multiple serializer.

Reply:
Good find, https://github.com/ansible/django-ansible-base/pull/351 tackles the root of the problem.

I'm pushing a commit here, but now that's just adding more tests for the most part and makes little functional difference.